state_of_css.yml

```
@@ -117,132 +117,132 @@ translations:
-      t: Often
+      t: Frequentemente
 
     - key: options.css_for_email.3
-      t: I focus mainly on CSS for email clients
+      t: Eu me concentro principalmente em CSS para clientes de e-mail
     - key: options.css_for_email.3.short
-      t: Mainly
+      t: Principalmente
 
     # what do you use CSS for?
     - key: options.what_do_you_use_css_for.marketing_sites
-      t: Marketing sites & landing pages
+      t: Sites de marketing e landing pages
     - key: options.what_do_you_use_css_for.design_systems
       t: Design systems
     - key: options.what_do_you_use_css_for.blogs
-      t: Blogs or other text-heavy sites
+      t: Blogs ou outros sites com muito texto
     - key: options.what_do_you_use_css_for.web_apps
-      t: Web apps
+      t: Aplicativos da Web
     - key: options.what_do_you_use_css_for.mobile_apps
-      t: Mobile apps
+      t: Aplicativos móveis
     - key: options.what_do_you_use_css_for.css_art
-      t: CSS art & illustrations
+      t: Arte e ilustrações CSS
     - key: options.what_do_you_use_css_for.emails
-      t: Emails
+      t: E-mails
     - key: options.what_do_you_use_css_for.printed_documents
-      t: Printed documents
+      t: Documentos impressos
 
     # CSS pain points
     - key: options.css_pain_points.browser_interoperability
-      t: Browser Compatibility
+      t: Compatibilidade do navegador
     - key: options.css_pain_points.browser_interoperability.description
-      t: Differences between Chrome, Safari, Firefox, etc.
+      t: Diferenças entre Chrome, Safari, Firefox, etc.
     - key: options.css_pain_points.interactions
-      t: Interactions
+      t: Interações
     - key: options.css_pain_points.interactions.description
-      t: Responding to user input or other events (scroll, hover, etc.). 
+      t: Respondendo à entrada do usuário ou outros eventos (rolagem, foco, etc.).
     - key: options.css_pain_points.architecture
-      t: Architecture & Maintenance
+      t: Arquitetura e Manutenção
     - key: options.css_pain_points.architecture.description
-      t: File organization, dead code elimination, refactoring, etc.
+      t: Organização de arquivos, eliminação de código morto, refatoração, etc.
     - key: options.css_pain_points.layout_positioning
-      t: Layout & Positioning
+      t: Layout e Posicionamento
     - key: options.css_pain_points.layout_positioning.description
-      t: Setting up layouts and making sure elements go where you want them to. 
+      t: Configurando layouts e certificando-se de que os elementos estão onde você deseja.
     - key: options.css_pain_points.scoping_specificity
-      t: Scoping & Specificity
+      t: Escopo e especificidade
     - key: options.css_pain_points.scoping_specificity.description
-      t: Dealing with the cascade, overriding styles, etc.
+      t: Lidando com a cascata, substituindo estilos, etc.
     - key: options.css_pain_points.responsive_design
-      t: Responsive Design
+      t: Design responsivo
```

Deve ser título de seções, por isso pode ser interessante manter maíscula

```suggestion
      t: Design Responsivo
```